style.css

```diff
@@ -0,0 +1,24 @@
+
+
+/* mobile first */
+
+body {
+    background-color: #74b9ff;
+}
+
+.barChart {
+  width: 80%;
+}
+.map {
+  width: 80%;
+}
+
+/* Desktop */
```

Comment:
desktop i think it's  min 1000 px
mobile<600
tablet 600-1000
desktop 1000+

What do you think ?

Reply:
The cut-offs I have been using are mobile:<480px, tablet <800, desktop 800+. But I supose the exact limits don't matter much as long as it looks good accross the board